app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ApplicationController < ActionController::API
+    end
```

Comment:
```suggestion
    class ApplicationController < ActionController::API; end
```

Since ruby 3 we can inline empty classes :) it doesn't change much, but just to let you know about it